src/components/TaskDetail.jsx

```diff
@@ -272,6 +287,73 @@ export function TaskDetail({ task, isOpen, onClose, onUpdate, onDelete, onAddLab
     onUpdate(task.id, { priority: newPriority });
   };
 
+  const handleAddSubtask = () => {
+    if (!newSubtask.trim()) return;
+    
+    const newSubtaskItem = {
+      id: crypto.randomUUID?.() || `${Date.now()}-${Math.random().toString(36).substr(2, 9)}-${Math.random().toString(36).substr(2, 9)}`,
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```javascript
id: crypto.randomUUID?.() || generateFallbackId(), // Add utility function for consistent ID generation
```
The handleAddSubtask function uses a complex fallback for generating IDs. Consider using a more reliable UUID generation method or extracting the ID generation logic into a utility function for consistency across the application.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;